testsuite/integration-deprecated/src/test/java/org/keycloak/testsuite/broker/AbstractKeycloakIdentityProviderTest.java

```diff
@@ -97,46 +97,6 @@ public void testDisabledUser() throws Exception {
         }
     }
 
-    @Test
-    public void testTemporarilyDisabledUser() {
-        KeycloakSession session = brokerServerRule.startSession();
-        setUpdateProfileFirstLogin(session.realms().getRealmByName("realm-with-broker"), IdentityProviderRepresentation.UPFLM_OFF);
```

Comment:
Removed in favor of AbstractBrokerTest#loginWithExistingUserWithBruteForceEnabled